### PR TITLE
Pgr maps integration

### PR DESCRIPTION
**Pull request by saurabh-egov**

### CODE REVIEW

File: packages/modules/pgr/src/pages/citizen/Create/Steps/SelectPincode.js [hunk 1]
```diff
@@ -8,11 +8,19 @@ const SelectPincode = ({ t, config, onSelect, value }) => {
...
   });
-
+  let isNextDisabled = pincode ? false : true;
   const [pincodeServicability, setPincodeServicability] = useState(null);
 
   function onChange(e) {
     setPincode(e.target.value);
+    
+    if(!e.target.value) {
+      console.log("changed")
```
Comment (saurabh-egov) [inline packages/modules/pgr/src/pages/citizen/Create/Steps/SelectPincode.js:18]: remove console logs

File: packages/react-components/src/atoms/LocationSearch.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import React, { useEffect } from "react";
 import { SearchIconSvg } from "./svgindex";
+import { Loader } from "@googlemaps/js-api-loader";
+
+//API key
+const key = globalConfigs.getConfig("GMAPS_API_KEY");
 
 const GetPinCode = (places) => {
-  console.log("Places addre component:", places.address_components);
+  console.log("Places address component:", places.address_components);
```
Comment (saurabh-egov) [inline packages/react-components/src/atoms/LocationSearch.js:9]: please remove console log

File: packages/react-components/src/atoms/LocationSearch.js [hunk 3]
```diff
@@ -110,6 +112,35 @@ const LocationSearch = (props) => {
           map.fitBounds(bounds);
         });
       };
+      const onMarkerDragged = (marker) => {
+        if (!marker) return;
+        console.log({ marker });
```
Comment (saurabh-egov) [inline packages/react-components/src/atoms/LocationSearch.js:117]: remove console

**Comment by saurabh-egov** [comment]
I cant see if you have taken care of the places showing only locality/ addresses rather than whole cities and maintaining boundary

**Comment by GaganLohia** [comment]
> I cant see if you have taken care of the places showing only locality/ addresses rather than whole cities and maintaining boundary please correct me if im wrong

Working on it

**Pull request merged**